app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -224,7 +224,7 @@ class RevisionCardFragmentTest {
   }
 
   @Test
-  fun testRevisionCardTestActivity_configurationChange_toolbarTitle_fractionSubtopicId1_isDisplayedCorrectly() { // ktlint-disable max-line-length
+  fun testRevisionCardTest_configChange_toolbarTitle_fractionSubtopicId1_IsDisplayedCorrectly() {
```

Comment:
Fix capitalization in front of ``IsDisplayed`` (each word starting after an ``_`` should be lowercase).